src/main/resources/explanations/challenge5_hint.adoc

```diff
@@ -0,0 +1,17 @@
+You can solve this challenge by the following steps:
+
+1. Check the status of the configmap in Git
+- Can you see where in git we stored the `secrets-config.yml`? If not, just do a search.
+- Take a look at the `Data` field: what can you find there?
+2. Ask nicely using Kubectl:
+- Make sure you have Kubectl installed as defined in the README.MD & make sure kubectl is configured to send commands to the right cluster.
+- Do `kubectl get configmap` . Here you see all the configmaps active in the namespace you are in, which is for this app normally `default` (unless otherwise specified by your administrator/trainer).
```

Comment:
```suggestion
- Do `kubectl get configmap`. Here you see all the configmaps active in the namespace you are in, which is for this app normally `default` (unless otherwise specified by your administrator/trainer).
```